Islandora-Devops/migrate_7x_claw
# Skip missing audit files

Resolves: https://github.com/Islandora-CLAW/CLAW/issues/1004

## How to test

- Try to migrate from 7x without this, you'll probably blow up.  Mine did because the basic image collection didn't have an audit ds.
- Pull in this PR
- `drush fim -y islandora_migrate_7x_claw_feature`
- You'll have to `migrate:reset islandora_audit_file` or perhaps `islandoa_audit_media` because they blew up on you.
- Migrate again.  This time it shouldn't blow up.  You'll probably still see errors on the console, but those will be fixed with https://github.com/Islandora-CLAW/islandora_demo/pull/14
